golang/pkg/crane/k8s/secret_store.go

```diff
@@ -0,0 +1,30 @@
+package k8s
+
+import "github.com/dyrector-io/dyrectorio/golang/internal/config"
+
+func (s *Secret) CheckPermissions() error {
+	// TODO (@m8vago, @nandor.magyar) implement
```

Comment:
The http get of the getSecret will throw 403 if there is any problem with the privileges in kubernetes, I don't think it is necessary here. Wouldn't it be better off in the normal Get as an error in case of a permission problem?

Reply:
This is used by the internal lib to check if we can replace a token, before we notify crux of the token replacement. It's necessary so we don't end up with the old token while crux thinks that the new one is in use.